Src/Extern/HYPRE/AMReX_Habec_3D_K.H

```diff
@@ -952,7 +952,7 @@ void habec_ijmat_eb (GpuArray<Real,27>& sten, Array4<Int> const& ncols,
                                                                                - gxyz};
             Array1D<Real,0,7> feb;
             for (int ii=0; ii<8; ii++) {
-                feb(ii) = -phig1(ii) * (ba(i,j,k) * beb(i,j,k) / dg);
+	      feb(ii) = -phig1(ii) * ( anorm * beb(i,j,k) / (dg*scaling) );
```

Comment:
It seems that the change is incomplete.  Additionally, we could also remove `ba` from the function parameter.